india_banking_connector/connectors/doctype/axis_bank_connector/axis_bank_connector.py

```
@@ -0,0 +1,396 @@
+		unique_id = "".join(re.findall(r"[0-9a-zA-Z]", payment_details.name))
+
+		if existing_payment_response := self.validate_duplicate_payments(
+			unique_id=unique_id,
+		):
+			return existing_payment_response
+
+		url = self.urls.make_payment
+		headers = self.headers
+		payload = self.get_encrypted_payload(method="make_payment")
+
+		response = requests.post(
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)